CONTRIBUTING.md

```diff
@@ -2,33 +2,27 @@
 
 Thank you for your interest in contributing to the `wikipron` codebase!
 
-## Setting Up A Development Environment
+This page assumes that you have already created a fork of the `wikipron` repo
+under your GitHub account and have the codebase available locally for
+development work. If you have followed
+[these steps](https://github.com/kylebgorman/wikipron#development),
+then you are all set.
 
-1. Create a fork of the `wikipron` repo on GitHub.
-2. Make sure you are in some sort of a virtual environment
-   (venv, virtualenv, conda, etc).
-3. Download and install the library in the "editable" mode
-   together with the core and dev dependencies within the virtual environment:
+## Working on a Feature or Bug Fix
```

Comment:
Sorry to be picky here but newline before this?

Reply:
Done.